### ISSUE TITLE
Pipette Tip Inventory

### ISSUE THREAD

**Issue by ocattau** [RobertsLab/resources#1415]
For the Finnpipette Multi-Channel the range is 50-300uL, so we have to use the 200uL as it is the only pipette tip that fits. My volume I need to pipette into every sample for the BSA assay is 250uL, so I need either 250uL or 300uL pipette tips. We don't have a larger multi-channel in lab. This is why I ordered https://www.usascientific.com/tipone-refills/p/tipone-refills I could also use 2x125uL to get to 250uL which would use twice as many tips. 

I also need to pipette 50uL very rapidly for the CS assay. I ordered the Eppendorf Combitips Advanced 2.5mL to fit with the repeater pipette https://www.pipette.com/0030089553-Eppendorf-Combitips-advanced-2-5-mL-Sterile-Individually-Wrapped-Tips-Green-100-per-Case If the tips don't come, my only option is to use the p200 or the Finnpipette Multi-Channel 300uL again with 200uL tips. 

If the tips I ordered don't come, I will need a ridiculous amount of 200uL (700 samples x 3 Multi-Channel 200uL tips)=2100 tips

I also need 700 p10 and p1000 for the rest of the assay that is not time sensitive

Question for Sam: What brand would you suggest for Thermo and Eppendorf pipette tips?

**Comment by kubu4** [comment]
> What brand would you suggest for Thermo and Eppendorf pipette tips?

I don't have any recommendations. Usually just go with cheapest filtered tips possible. We've been using USA Scientific because they offer the refill packs for filtered tips, which helps reduce plastic usage. But, you're free to order whatever will work for you.

**Comment by kubu4** [comment]
I glanced at the assay manual (I'm assuming you're using the Bio-Rad Quick Start Bradford Protein Assay) and an alternative is to add the 1x Dye Reagent to the plate _before_ adding the standards/samples. This way, you wouldn't have to change tips _at all_. You'd only have to a single set of 8 (or 12?) tips on a multi-channel to load the entire plate.

After you've loaded the 1x Dye Reagent, then you'd add your samples. Of course, you'd have to shake the plate a bit for mixing purposes. I believe the plate reader has this functionality built-in. Alternatively, we should have an orbital shaker in the lab to do this as well.

And, honestly, I would prefer this approach regardless. When pipetting small volumes of sample (5uL, per the manual), it's always best to be able to add them to existing liquid instead of bare plastic (e.g. microfuge tube, plate well), as the surface tension between the sample in the tip and the liquid in the tube/plate helps to ensure complete dispensing of the small volume from the pipette tip.

**Comment by ocattau** [comment]
Thats a good point Sam, in that case I would load the 250uL first then use the 10uL for each well.

**Comment by ocattau** [comment]
Just spoke to a USAscientific rep and they are out of ALL pipette tips...wild https://www.usascientific.com/tipone-refills/p/tipone-refills

**Comment by kubu4** [comment]
This is getting a bit off-topic, but I noticed this:

>  250uL first then use the 10uL for each well.

If you're maintaining the recommended ratio of dye:sample listed in the manual (50:1), then shouldn't your dye volume be changed to 500uL? Also, have you made the same change to the standards ratio (i.e. 10uL of standards to each well)? I think the latter is the bigger concern, rather than the overall dye:sample/standard ratio (although, I'd probably keep the ratio if possible - unless you've tested/compared 5uL sample/standard with 250uL dye and 10uL sample/standard with 250uL dye and gotten same results).

Also, on the standards front, have you been diluting them in CS buffer?

Also, I noticed this in [your protocol](https://docs.google.com/document/d/1CZtU6QTkBIhtNHrl0dIxgEd1ubukJDFf64_rAckiHkY/edit) (Google Doc):

> 6. Dilute protein standard (BSA and gamma-globulin) in triplicate.

Are you using two sets of standards on each plate? I'd think you'd only need to use a single standard (looks like manual recommends the gamma-globulin)

Sorry for all the questions! Curiosity got the best of me.
